examples/connatix-ps-player.amp.html

```diff
@@ -0,0 +1,109 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <title>amp-connatix-ps-player example</title>
+  <link rel="canonical" href="amps.html">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <style amp-custom>
+      body {
+        max-width: 1600px;
+        padding: 12px;
+      }
+    </style>
+  <script async custom-element="amp-connatix-ps-player" src="https://cdn.ampproject.org/v0/amp-connatix-ps-player-0.1.js"></script>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+</head>
+<body>
+  <h2>Connatix AMP examples</h2>
+  <h3>Connatix Player Responsive</h3>
+  <p>fdsfsdfsd</p>
+  <p>fdsfsdfsd</p>
+  <p>fdsfsdfsd</p>
+  <p>fdsfsdfsd</p>
```

Comment:
Example pages need to look proper :)) remove filler stuff

Reply:
I leave them for testing floating off. I'll remove them and added them back after js api setting is done.